src/ExportPostman.php

```diff
@@ -31,15 +29,13 @@ public function __construct(Router $router)
 
     public function handle(): void
     {
-        $structured = $this->option('structured') ?? false;
-        $baseUrl = $this->option('base-url') ?? 'https://api.example.com/';
         $bearer = $this->option('bearer') ?? false;
 
         $this->routes = [
             'variable' => [
                 [
                     'key' => 'base_url',
-                    'value' => $baseUrl,
+                    'value' => config('api-postman.base_url'),
```

Comment:
Let's load the config file into it's own property in the constructor. 😃

Reply:
Done